README.md

```diff
@@ -130,7 +133,42 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
 
 6. Open `http://<IP OF YOUR SERVER>:3000` in your browser.
 
+## Carefully in Windows
+
+If you want to run Judge0 API on Windows, you have to change file `docker-compose.yml` .
+
+This is sample about `docker-compose.yml` in Windows.  We had run it succeed on Windows10 via `PowerShell`.
```

Comment:
You could create new `docker-compose.windows.yml` in repo and just reference to it. I like the part about successful run on Windows 10 :smile:, but PowerShell comment is not necessary if you can also run it via good old CMD.